vmhost/contexts/async_test.go

```diff
@@ -835,7 +835,7 @@ func createCallbackInput(t *testing.T, lastTransfer *vmcommon.OutputTransfer) (*
 	return createCallbackInputWithVMOutput(t, vmOutput, lastTransfer)
 }
 
-func createCallbackInputWithVMOutput(t *testing.T, vmOutput *vmcommon.VMOutput, lastTransfer *vmcommon.OutputTransfer) (*vmcommon.ContractCallInput, *vmcommon.ContractCallInput, uint64) {
+func createCallbackInputWithVMOutput(t *testing.T, vmOutput *vmcommon.VMOutput, _ *vmcommon.OutputTransfer) (*vmcommon.ContractCallInput, *vmcommon.ContractCallInput, uint64) {
```

Comment:
what is the point of an ignored function argument?

Reply:
fixed the linter issue.